image-streams/imagesignature.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ImageSignature
+metadata:
+ name: sha256:29f5d56d12684887bdfa50dcd29fc31eea4aaf4ad3bec43daf19026a7ce69912@imagesignaturetest2
```

Comment:
Invalid value: "sha256:fcab41c08affc3f67d598dad7a7d928a47c8a65a8b849dc3d9f64b29c75ddfdb": name must be of format <imageName>@<signatureName>